go/oppobloom/oppobloom.go

```diff
@@ -17,8 +17,10 @@ import (
 )
 
 type Filter struct {
-	array []*[]byte
-	sizeMask uint32
+	array      []*[]byte
```

Comment:
There's a bunch of gofmt changes in this code (it seems to predate my auto-gofmt set up!), so I've pushed up a gofmt change separately that now conflicts with this PR in parts.

Sorry but thanks for the heads up!